### ISSUE TITLE
Rule: space-after-comma

### ISSUE THREAD

**Issue by goloroden** [eslint/eslint#968]
There should be a rule that checks whether there's a space after a comma, unless the comma is the last character in the line:

```
var foo = { bar: 42, baz: 23 }; // This is fine
var foo = { bar: 42,baz: 23 };  // This raises an error
```

The rule's name could be `space-after-comma` and it might be enabled by default.

What do you think?
